test/send_recv.c

```diff
@@ -214,18 +214,21 @@ static int do_send(void)
 	ret = io_uring_wait_cqe(&ring, &cqe);
 	if (cqe->res == -EINVAL) {
 		fprintf(stdout, "send not supported, skipping\n");
-		close(sockfd);
-		return 0;
-	}
+		goto err;
+        }
```

Comment:
I missed this. One more here, the indentation should use tabs, not spaces.

Reply:
Done